backend/src/main/java/de/kaiheinrich/projectplace/dto/ProfileDto.java

```diff
@@ -0,0 +1,18 @@
+package de.kaiheinrich.projectplace.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+public class ProfileDto {
+
+    private String name;
+    private String age;
```

Comment:
Use an `Instant` and store date of birth, so that you don't have to update age once every year.